src/components/widgets-common/chart-threshold/chart-threshold.component.coffee

```diff
@@ -165,7 +161,7 @@ module.component('chartThreshold', {
       widgetHistParams = ctrl.widget.metadata && ctrl.widget.metadata.hist_parameters
       # Widget histParams are YTD by default (when undefined on metadata),
       # therefore in the past by default
-      ctrl.disabled = _.isEmpty(widgetHistParams) || moment(widgetHistParams.to) <= moment().startOf('day')
+      ctrl.disabled = widgetHistParams? && moment(widgetHistParams.to) <= moment.utc().startOf('day')
```

Comment:
if `widgetHistParams` is equal to `undefined` or `null`, `ctrl.disabled` will equal `false`. Is this intended?

Reply:
yes: I disable it only if I have hist parameters defined so that the time period is in the past.
the default time period is no longer YTD but -3m / +1m